### DISCUSSION TITLE
Custom Domains is already used in a SimpleLogin mailbox

### DISCUSSION THREAD

**Discussion by EthraZa** [simple-login/app#674] [Q&A] [answered]
Hi.
Not sure if someone can help me here with a problem at simplelogin.io but, since no one responds at hi@simplelogin.io, I'm taking my chances.

1- I have created an account at simplelogin.io with my user@DOMAIN-A.tld.
2- I tryed to add the custom domain [DOMAIN-A.tld](url) and it told me I can't use the same login domain as a custom domain.
3- I have deleted the account  user@DOMAIN-A.tld.
3- Created a new account with user@DOMAIN-B.tld.
4- When try to add the custom domain [DOMAIN-A.tld](url) it now says: **DOMAIN-A.tld already used in a SimpleLogin mailbox**

Now I see that my problem probably was step 3. I now saw that I could have just change the login email in the options, but now is too late, I deleted the account already.

There is some way to add my custom domain? There is some other, more responsive, contact channel to simplelogin.io?

Thank you in advance.

**Comment by nguyenkims** [comment, marked as answer]
@EthraZa we have responded 2 days ago, maybe the response went into your Spam folder? Otherwise we have investigated and the error is correct, there's another user who uses DOMAIN-A.tld as their mailbox address in SimpleLogin. As you own this domain, I guess you can know who this person is and maybe can ask them to change their mailbox address.

**Reply by EthraZa** [reply]
Hi.

I have searched and there is no response in spam or anywhere... that's is strange.
But, can you tell me what is the left part of the user (???@DOMAIN-A.tld)?

Tks

**Reply by nguyenkims** [reply]
Hi we have replied again today so that's strange that the email doesn't reach you. We haven't received any bounce report so I wonder if this is possible that your email service "silently" drops the email?

**Reply by EthraZa** [reply]
No dice. 
I just sent you a new email from my personal Gmail account (same used here at GH as the user), mentioning this conversation. Please, answer there so we can see if I receive. There, in CC, I have 2 Gmail accounts and 1 in my own mail server. Now I got receive the answer somewhere. :)

Thank you in advance.

**Reply by EthraZa** [reply]
I got you response via email and followed there. Also, I found out why I didn't get the other response. All the mess is my fault. Sorry about that.

Let me note here that your support seems indeed very quick, responsive and helpful. SimpleLogin service looks premium in every aspect.

Thank you.

**Reply by nguyenkims** [reply]
Thanks for your kind words! Glad that all issues seem to be solved now :).
